src/main/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGenerator.java

```diff
@@ -51,6 +63,33 @@ protected static String createRepoURI(String url, String projectName) {
         }
     }
 
+    protected static Link createLink(String linkType, String linkedEiffelEventType, File pluginDirectoryPath, final String projectName,
+            final String tableColumnName) {
```

Comment:
Should we really know about tables when creating a link?

Reply:
Broke the event fetching into its own method in upcoming change :)